### PR TITLE
Server Initialization

### LINKED ISSUE #59: Initiate Web App Server
Initiate a web app server with basic scaffolding done.

### PR DESCRIPTION
**Pull request by sweezy793**
### Checklist:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you successfully ran tests with your changes locally?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

This PR is for the initialization of the web server, it includes basic scaffolding and temporary endpoints. 

### Issues Solved

Fixes #59

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
Hi! Thanks for PR.
Make sure you have checked points in PR Template correctly.

(Powered by [GitMate.io](https://gitmate.io))

File: server/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```
Comment (aashutoshrathi) [inline server/.gitignore]: A newline at EOF needed.

File: server/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "Server for HackerRank Test Case Generator",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "server": "nodemon index.js"
+  },
+  "author": "",
```
Comment (aashutoshrathi) [inline server/package.json]: Ahh.. you can always fill this.

File: server/.gitignore (hunk 1 above)
Comment (sweezy793) [reply server/.gitignore]: Will do!

File: server/package.json (hunk 2 above)
Comment (sweezy793) [reply server/package.json]: Ohh, sure sure :)

**Comment by sweezy793** [comment]
> Plus where would you like for me to host this server? Glitch or Heroku?

Heroku would be better

**Comment by sweezy793** [comment]
@aashutoshrathi Please review the new changes

**Comment by aashutoshrathi** [comment]
API is live at: https://tc-gen.herokuapp.com/

**Pull request merged**